go-controller/pkg/node/openflow_manager.go

```diff
@@ -239,7 +239,30 @@ func (c *openflowManager) updateBridgeFlowCache(hostIPs []net.IP, hostSubnets []
 	}
 	dftFlows = append(dftFlows, dftCommonFlows...)
 
-	c.updateFlowCacheEntry("NORMAL", []string{fmt.Sprintf("table=0,priority=0,actions=%s\n", util.NormalAction)})
+	checkCmd := []string{
+		"get",
+		"Open_vSwitch",
+		".",
+		"external_ids:ovn-encap-ip",
+	}
+	encapIP, _, err := util.RunOVSVsctl(checkCmd...)
+	if err != nil {
+		return err
+	}
+	encapIP = strings.TrimSuffix(encapIP, "\n")
+
+	if config.OvnKubeNode.Mode == types.NodeModeDPU {
+		c.updateFlowCacheEntry("NORMAL", []string{
+			fmt.Sprintf("table=0,priority=0,actions=%s\n", util.NormalAction),
+			fmt.Sprintf("table=0,priority=999,udp,tp_dst=67,actions=%s\n", util.NormalAction),
+			fmt.Sprintf("table=0,priority=999,udp,nw_dst=%s,tp_dst=6081,actions=LOCAL\n", encapIP),
+			fmt.Sprintf("table=0,priority=998,ip,nw_dst=%s,actions=%s\n", encapIP, util.NormalAction),
+		})
```

Comment:
why?

Reply:
The 67 port is to allow dhcp traffic to be passed to the PF (physical function).

6081 is to send the geneve traffic to the bridge (local port).

The last one which matches the encap ip is to be able to ping the local port from the PF.

If it is helpful I can add the bridge configuration and the IP management.